src/shared/cfg.py

```diff
@@ -291,20 +291,17 @@ def LOCAL_FLASK_PORT(self):
         """
         return self("LOCAL_FLASK_PORT", 5000, cast=int)
 
+    @property
+    def DYNALITE_URL(self):
+        return self("DYNALITE_URL", "http://127.0.0.1:8000")
```

Comment:
A default item is debatable here.  Furthermore, no docstring was added as I can see no code documentation generation tool being used and the field naming is self evident.

Reply:
heh; then we should delete them all; i did that so things like pylint wont complain; self-evident agreed though

Reply:
it's also bad practice to depart from prevailing coding style (in this case docstrs on all properties) without addressing the issue raise with every one of them.  Leads to messy code and lack of consistency throughout

Reply:
In another work stream, I propose to add a pylint pre-commit hook.  As an option to that hook, I propose adding an option to `--disable` docstring warnings.  It is easy enough to add in.